packages/js-runtime/typescript/resolver.ts

```diff
@@ -80,7 +80,7 @@ function isUnresolvedModuleOk(
 }
 
 function resolveSpecifier(specifier: string, from: Source): string {
-  if (specifier[0] === "." || specifier[1] === "/") {
```

Comment:
I think this should be `specifier[0] === "." && specifier[1] === "/"`, want to capture `./relative/imports/`, though this looks like it's missing some scenarios (though `../` is tested elsewhere) hm